# vendor diff, original (main tree, uncommitted) -> vendored (this task)
# generated 2026-08-29T06:17:47Z
=== PT.agda ===
@@ -18,7 +18,7 @@
 open import Base.Truth
 open import Base.Classical using ( LEM )
 
-module LJ-1-746-SPLIT.runs.PT {ℓ : Level} (lem : LEM (ℓ-suc ℓ)) where
+module LJ-1-746-SPLIT-SPLIT.runs.PT {ℓ : Level} (lem : LEM (ℓ-suc ℓ)) where
 
 open import FOL.Syntax using ( var; _∧̇_ )
 open import FOL.Manipulation.Parameters using ( countFo; countTm )
@@ -30,8 +30,8 @@
 open import LJ-1-732.runs.Amb4a {ℓ} lem
 open import LJ-1-732.runs.Amb5 {ℓ} lem
 open import LJ-1-732.runs.Amb6 {ℓ} lem
-open import LJ-1-746-SPLIT.runs.Amb7 {ℓ} lem
-open import LJ-1-746-SPLIT.runs.EraseIrr {ℓ} lem
+open import LJ-1-746-SPLIT-SPLIT.runs.Amb7 {ℓ} lem
+open import LJ-1-746-SPLIT-SPLIT.runs.EraseIrr {ℓ} lem
 
 open TruthAlgebra (hPropAlgebra (ℓ-suc ℓ))
 open hPropStructure 𝒮ᵥ
=== Amb7.agda ===
@@ -17,7 +17,7 @@
 open import Base.Truth
 open import Base.Classical using ( LEM )
 
-module LJ-1-746-SPLIT.runs.Amb7 {ℓ : Level} (lem : LEM (ℓ-suc ℓ)) where
+module LJ-1-746-SPLIT-SPLIT.runs.Amb7 {ℓ : Level} (lem : LEM (ℓ-suc ℓ)) where
 
 open import FOL.ZFStructure using ( module hPropStructure )
 open import FOL.Syntax using ( Formula )
=== EraseIrr.agda ===
@@ -36,7 +36,7 @@
 open import Cubical.Data.Nat using ( snotz )
 import Cubical.Data.Empty as Empty
 
-module LJ-1-746-SPLIT.runs.EraseIrr {ℓ : Level} (lem : LEM (ℓ-suc ℓ)) where
+module LJ-1-746-SPLIT-SPLIT.runs.EraseIrr {ℓ : Level} (lem : LEM (ℓ-suc ℓ)) where
 
 module Irr (K : Type (ℓ-suc ℓ)) where
   private
